packages/x-data-grid/src/models/gridExport.ts

```diff
@@ -40,6 +40,12 @@ export interface GridFileExportOptions<Api extends GridApiCommon = GridApiCommun
    * @returns {GridRowId[]} The list of row ids to export.
    */
   getRowsToExport?: (params: GridGetRowsToExportParams<Api>) => GridRowId[];
+  /**
+   * If `true`, the formulas in the cells will be escaped.
+   * It is recommended to set this to `true` for security reasons.
+   * @default false
+   */
+  escapeFormulae?: boolean;
```

Comment:
`escapeFormulas`? From what I can read online formulas seems to be the most popular plural for formula.

Reply:
I followed the same naming that `exceljs` used:

https://github.com/mui/mui-x/blob/760ea7e844b2d985d78a9a45295ca1b050defb11/packages/x-data-grid-premium/src/hooks/features/export/serializer/excelSerializer.ts#L123

But I actually prefer `escapeFormulas`, I'll update 👍🏻